diff --git a/CMakeLists.txt b/CMakeLists.txtindex 78cec86..d374b5f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -106,21 +106,15 @@ include(FetchContent)
 
 FetchContent_Declare(
   repo-common
-  GIT_REPOSITORY https://github.com/triton-inference-server/common.git
-  GIT_TAG ${TRITON_COMMON_REPO_TAG}
-  GIT_SHALLOW ON
+  SOURCE_DIR @repoCommon@
 )
 FetchContent_Declare(
   repo-core
-  GIT_REPOSITORY https://github.com/triton-inference-server/core.git
-  GIT_TAG ${TRITON_CORE_REPO_TAG}
-  GIT_SHALLOW ON
+  SOURCE_DIR @repoCore@
 )
 FetchContent_Declare(
   repo-backend
-  GIT_REPOSITORY https://github.com/triton-inference-server/backend.git
-  GIT_TAG ${TRITON_BACKEND_REPO_TAG}
-  GIT_SHALLOW ON
+  SOURCE_DIR @repoBackend@
 )
 FetchContent_MakeAvailable(repo-common repo-core repo-backend)
